system/src/ble_control_request_channel.h

```diff
@@ -64,6 +64,16 @@ class BleControlRequestChannel: public ControlRequestChannel {
     virtual void freeRequestData(ctrl_request* ctrlReq) override;
     virtual void setResult(ctrl_request* ctrlReq, int result, ctrl_completion_handler_fn handler, void* data) override;
 
+    int setProvSvcUuid(hal_ble_uuid_t* svcUuid);
+    int setProvTxUuid(hal_ble_uuid_t* txUuid);
+    int setProvRxUuid(hal_ble_uuid_t* rxUuid);
+    int setProvVerUuid(hal_ble_uuid_t* verUuid);
+
+    hal_ble_uuid_t getBleCtrlSvcUuid();
+
+    int setCompanyId(uint16_t companyId);
```

Comment:
The declaration of `setCompanyId()` and `getCompanyId()` in this class are no more needed.